src/content/developers/docs/consensus-mechanisms/pos/index.md

```diff
@@ -71,16 +71,16 @@ As Vlad Zamfir put it, this is like a miner participating in a 51% attack, causi
 
 ## Proof-of-stake and security
 
-The threat of a [51% attack](https://www.investopedia.com/terms/1/51-attack.asp) still exists in proof-of-stake but it's even more risky for the attackers. To do so, you'd need to control 51% of the staked ETH. Not only is this a lot of money but it would probably cause ETH's value to drop. There's very little incentive to destroy the value of a currency you have a majority stake in. There are stronger incentives to keep the network secure and healthy.
+The threat of a [51% attack](https://www.investopedia.com/terms/1/51-attack.asp) still exists in proof-of-stake, but it's even more risky for the attackers. To do so, you'd need to control 51% of the staked ETH. Not only is this a lot of money, but it would probably cause ETH's value to drop. There's very little incentive to destroy the value of a currency you have a majority stake in. There are stronger incentives to keep the network secure and healthy.
 
-Stake slashings, ejections, and other penalties, coordinated by the beacon chain, will exist to prevent other acts of bad behaviour. Validators will also be responsible for flagging these incidents.
+Stake slashings, ejections, and other penalties, coordinated by the beacon chain, will exist to prevent other acts of bad behavior. Validators will also be responsible for flagging these incidents.
```

Comment:
https://github.com/ethereum/ethereum-org-website/issues/2786
Hah, note there is an open issue about British/American English inconsistencies... It's fine here, but before going much further with this specific type of change, let's see what Ryan thinks